src/App/Pages/Accounts/LoginPasswordlessViewModel.cs

```diff
@@ -63,8 +70,30 @@ public LoginPasswordlessDetails LoginRequest
             }
         }
 
+        public void StopRequestTimeUpdater() => _requestTimeCts?.Cancel();
+
+        private void StartRequestTimeUpdater()
+        {
+            _requestTimeCts?.Cancel();
+            _requestTimeCts = new CancellationTokenSource();
+            _requestTimeTask = new TimerTask(_logger, UpdateRequestTime, _requestTimeCts).RunPeriodic(TimeSpan.FromMinutes(5));
```

Comment:
Could you extract the `5` as a constant in this class? so if one wants to change the frequency they don't have to read the code and just update the constant.
And since you're here, could you change the parameter of the `TimerTask` to be the `CancellationToken` instead of the `CancellationTokenSource`? There is no need to cancel the request from inside the `TimerTask` so the token should suffice.
[Quoting](https://stackoverflow.com/a/54386551) the great Jon Skeet:

> The rule of thumb is:
> - Any code that needs to cancel the token should have access to the CancellationTokenSource
> - Any code that needs to respond to cancellation should have access to the CancellationToken